rules/windows/credential_access_iis_apppoolsa_pwd_appcmd.toml

```
@@ -21,12 +21,12 @@ risk_score = 73
 rule_id = "0564fb9d-90b9-4234-a411-82a546dc1343"
 severity = "high"
 tags = ["Elastic", "Windows"]
-type = "query"
+type = "eql"
 
 query = '''
-event.category:process AND event.type:(start OR process_started) AND
-  (process.name:appcmd.exe OR process.pe.original_file_name:appcmd.exe) AND
-  process.args:(/[lL][iI][sS][tT]/ AND /\/[tT][eE][xX][tT]\:[pP][aA][sS][sS][wW][oO][rR][dD]/)
+process where event.type in ("start", "process_started") and
+   (process.name : "appcmd.exe" or process.pe.original_file_name : "appcmd.exe") and 
```

original_file_name compares _should_ stay case sensitive

```suggestion
   (process.name : "appcmd.exe" or process.pe.original_file_name == "appcmd.exe") and 
```